libbeat/processors/actions/json_decode_fields.go

```diff
@@ -0,0 +1,91 @@
+package actions
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/processors"
+)
+
+type jsonDecodeFields struct {
+	Fields []string
+}
+
+func init() {
+	processors.RegisterPlugin("json_decode_fields",
```

Comment:
Naming-wise I'd make it `decode_json_fields`. The other processors put the verb first.